src/components/RegisterForm/BarCodeScanner.vue

```diff
@@ -85,7 +93,7 @@ const stop = () => {
 const onResume = (e: Event) => {
   if (
     !(e.target as HTMLVideoElement).paused &&
-    selectInput.value !== undefined
+    serectedDevice.value !== undefined
```

Comment:
今は`undefined`にならず`''`にしてるから
```suggestion
    !serectedDevice.value
```